### ISSUE TITLE
Toolbox wps

### ISSUE THREAD

**Issue by ebocher** [orbisgis/orbisgis#1045]
The WPS toolbox doesn't work on Windows and Mac OS. The OrbisGIS application crashes.

**Comment by ebocher** [comment]
Some error messages

```java
Got java.lang.NullPointerException when invoking "public void org.orbisgis.mapuce.WpsScriptsPackage.activate()" in: class org.orbisgis.mapuce.WpsScriptsPackage
```

**Comment by ebocher** [comment]
OS :  Windows 7 64bit.
JAVA 8 (1.8.0_91-b14)...

```java
Enable Immediate component: org.orbisgis.progressgui.JobListBar
Enable Immediate component: org.orbisgis.progressgui.WatchExecutorCommand
Enable Delayed component: org.orbisgis.progressgui.WatchExecutorService
Satisfied: Delayed component: org.orbisgis.progressgui.WatchExecutorService
Satisfied: Immediate component: org.orbisgis.progressgui.WatchExecutorCommand
Enable Delayed component: org.orbisgis.scp.SQLLanguageSupport
Enable Delayed component: org.orbisgis.scp.ScriptSplitterFactoryImpl
Satisfied: Delayed component: org.orbisgis.scp.ScriptSplitterFactoryImpl
Enable Factory component: org.orbisgis.sqlconsole.SQLConsole
Enable Delayed component: org.orbisgis.sqlconsole.MainWindowMenu
Enable Immediate component: org.orbisgis.sqlconsole.SQLConsoleFactory
Enable Immediate component: org.orbisgis.tablegui.impl.TableEditorFactory
Enable Delayed component: org.orbisgis.tablegui.impl.DBTreeOpenTable
Enable Delayed component: org.orbisgis.tablegui.impl.GeoCatalogMenu
Enable Delayed component: org.orbisgis.tablegui.impl.TocMenu
Enable Delayed component: org.orbisgis.view.toc.Toc
Enable Immediate component: org.orbisgis.wkgui.gui.WorkspaceSelectionDialog
Satisfied: Immediate component: org.orbisgis.wkgui.gui.WorkspaceSelectionDialog
Enable Immediate component: org.orbisgis.wpsclient.WpsClientImpl
Enable Immediate component: org.orbisgis.wpsservicescripts.OrbisGISWpsScriptPlugin
Enable Delayed component: org.orbisgis.wpsservice.LocalWpsServerImplementation
Data source available H2 JDBC Driver version 1.4.188 (2015-08-01)
Satisfied: Immediate component: org.orbisgis.logwriter.WorkspaceLogConfigGenerator
Satisfied: Delayed component: org.orbisgis.corejdbc.DataSourceService
Satisfied: Factory component: org.orbisgis.sqlconsole.SQLConsole
Satisfied: Immediate component: org.orbisgis.sqlconsole.SQLConsoleFactory
Satisfied: Delayed component: org.orbisgis.scp.SQLLanguageSupport
Satisfied: Delayed component: org.orbisgis.corejdbc.internal.DataManagerImpl
Satisfied: Immediate component: org.orbisgis.h2triggersosgi.EventListenerService
Satisfied: Delayed component: org.orbisgis.wpsservice.LocalWpsServerImplementation
Satisfied: Delayed component: org.orbisgis.dbjobs.service.DriverFunctionContainerImpl
Satisfied: Immediate component: org.orbisgis.geocatalog.impl.Catalog
Satisfied: Delayed component: org.orbisgis.geocatalogtree.impl.CatalogPanel
Satisfied: Delayed component: org.orbisgis.mainframe.impl.MainFrame
Satisfied: Immediate component: org.orbisgis.progressgui.JobListBar
Reading repository contents in 0 ms
Satisfied: Delayed component: org.orbisgis.docking.impl.DockingManagerImpl
Satisfied: Immediate component: org.orbisgis.wpsclient.WpsClientImpl
Satisfied: Immediate component: org.orbisgis.wpsservicescripts.OrbisGISWpsScriptPlugin
Warning, because of the H2 configuration, the toolbox won't be able to run more than one process at the same time.
Try to use the following setting for H2 : 'MVCC=TRUE; LOCK_TIMEOUT=100000; MULTI_THREADED=TRUE'
Got java.lang.IllegalArgumentException: Illegal character in opaque part at index 2: C:\Users\serge\AppData\Local\Temp\wpsprocess2061437250443696649.groovy:input:Input_spatial_data when invoking "public void org.orbisgis.wpsservicescripts.OrbisGISWpsScriptPlugin.activate()" in: class org.orbisgis.wpsservicescripts.OrbisGISWpsScriptPlugin
Satisfied: Immediate component: org.orbisgis.wpsclient.WpsClientImpl
Satisfied: Delayed component: org.orbisgis.docking.impl.edition.EditorManagerImpl
Satisfied: Immediate component: org.orbisgis.tablegui.impl.TableEditorFactory
Satisfied: Delayed component: org.orbisgis.tablegui.impl.DBTreeOpenTable
Satisfied: Delayed component: org.orbisgis.tablegui.impl.GeoCatalogMenu
Satisfied: Delayed component: org.orbisgis.tablegui.impl.TocMenu
Satisfied: Delayed component: org.orbisgis.sqlconsole.MainWindowMenu
Satisfied: Delayed component: org.orbisgis.mapeditor.map.MapEditor
Satisfied: Delayed component: org.orbisgis.view.toc.Toc
Satisfied: Delayed component: org.orbisgis.mapeditor.map.toolbar.DrawingToolBar
Satisfied: Immediate component: org.orbisgis.docking.impl.DockingLayoutLoader
Rendering done in 0 seconds
Rendering done in 0 seconds 
```
